api/src/main/java/io/strimzi/api/kafka/model/kafka/cruisecontrol/KafkaAutoRebalanceModeBrokers.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.kafka.cruisecontrol;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.common.Constants;
+import io.strimzi.api.kafka.model.common.UnknownPropertyPreserving;
+import io.strimzi.api.kafka.model.rebalance.KafkaRebalanceMode;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents a mode related to an auto-rebalancing operation operation in progress or queued,
+ * together with the list of brokers' IDs involved in the rebalancing itself
+ */
+@Buildable(
+        editableEnabled = false,
+        builderPackage = Constants.FABRIC8_KUBERNETES_API
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"mode", "brokers"})
+@EqualsAndHashCode
+@ToString
+public class KafkaAutoRebalanceModeBrokers implements UnknownPropertyPreserving {
```

Comment:
So, this is for status, right? Would it make sense to have all the Status classes called accordingly to make it more clear? E.g. `KafkaAutoRebalanceStatusBrokers` or something like that?